### PR TITLE
Merge from stable

### PR DESCRIPTION
**Pull request by kasper-piskorski**

### CODE REVIEW

**Comment by kasper-piskorski** [comment]
!rtg

File: grakn-core/src/main/java/ai/grakn/util/Schema.java [hunk 1]
```diff
@@ -59,8 +59,8 @@ private Schema() {
         PLAYS("plays"),
         HYPOTHESIS("hypothesis"),
         CONCLUSION("conclusion"),
-        ROLE_PLAYER("role-player"),
-        RESOURCE("resource"),
+        SHORTCUT("shortcut"),
```
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/util/Schema.java]: This should stay `ROLE_PLAYER`

File: grakn-factory/janus-factory/src/main/resources/indices-edges.properties [hunk 2]
```diff
@@ -16,5 +16,5 @@
 # along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
 #
 
-role-player=RELATIONSHIP_TYPE_LABEL_ID,ROLE_LABEL_ID
-resource=RELATIONSHIP_TYPE_LABEL_ID
\ No newline at end of file
+shortcut=RELATIONSHIP_TYPE_LABEL_ID,ROLE_LABEL_ID
```
Comment (fppt) [inline grakn-factory/janus-factory/src/main/resources/indices-edges.properties]: This should also stay role-player

**Comment by fppt** [comment]
@kasper-piskorski you have some merge errors

**Pull request merged**